lib/board.rb

```diff
@@ -0,0 +1,189 @@
+class Board
+  WIN_COMBINATIONS=[[0,1,2],[3,4,5],[6,7,8],[0,4,8],[2,4,6],[1,4,7],[0,3,6],[2,5,8]] #defines all of the possible combinations to win
+  attr_accessor :cells
+  def reset! #resets the @cells which empties the board
+    @cells=[" "," "," "," "," "," "," "," "," "]
+  end
+    
+  def cells #is the board 
```

Comment:
With `attr_accessor :cells`, this method is not needed. `attr_accessor` is a macro that automatically adds both the setter method and the getter method to your class for the selected attribute. In this case, both
```
def cells
  @cells
end
```
and
```
def cells=(cells)
  @cells = cells
end
```
have been included in your class through the `attr_accessor`.